chromium_src/third_party/blink/renderer/core/frame/navigator_concurrent_hardware.cc

```diff
@@ -0,0 +1,57 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "third_party/blink/renderer/core/frame/navigator_concurrent_hardware.h"
+
+#include "base/compiler_specific.h"
+#include "base/system/sys_info.h"
+#include "brave/third_party/blink/renderer/farbling/brave_session_cache.h"
+#include "third_party/abseil-cpp/absl/random/random.h"
+
+namespace brave {
```

Comment:
why is this in brave instead of an anonymous namespace?

Reply:
This file shouldn't be here -- it's a merge error that I will fix